src/iop/highlights.c

```diff
@@ -85,13 +85,25 @@ typedef enum dt_atrous_wavelets_scales_t
   WAVELETS_10_SCALE = 9, // $DESCRIPTION: "2048 px (insanely slow)"
 } dt_atrous_wavelets_scales_t;
 
+typedef enum dt_recovery_mode_t
+{
+  RECOVERY_MODE_OFF = 0,    // $DESCRIPTION: "off" 
```

Comment:
Add prefix `DT_` to all values here.

And yes I know that this rule is not always followed, even in this module I see now that `enum dt_atrous_wavelets_scales_t` does not have this prefix. I can do this change after this PR is merge if you want, no problem, just noticing.

Reply:
ok, fixed that also for the wavelets

Reply:
ok